lib/Minilla/Project.pm

```diff
@@ -524,16 +524,16 @@ sub extract_git_info {
     if ( my $registered_url = `git config --get remote.origin.url` ) {
         $registered_url =~ s/\n//g;
         # XXX Make it public clone URL, but this only works with github
-        if ($registered_url !~ m{^file://} && $registered_url =~ /github\.com/) {
-            my ($user, $repo) = $registered_url =~ m{
-                github\.com
+        if ($registered_url !~ m{^file://} && $registered_url =~ /(?:github|gitlab)\.com/) {
+            my ($gitService, $user, $repo) = $registered_url =~ m{
```

Comment:
gitService -> git_service

Reply:
I've switched the variable name to the rest of the codebase style as suggested.